docs/diagrams/ConsultationClassDiagram.puml

```diff
@@ -1,20 +1,36 @@
-@startuml
+@startuml Consultation
 hide circle
 skinparam classAttributeIconSize 0
 
-Class Consultation {
+class Consultation {
```

Comment:
I thought class was supposed to be capitalised? With Reference to ModelClassDiagram

Reply:
oh, I was referencing to dinghan's class diagram, must have been mistaken, will change both for standardisation!